modules/telco-upgrade-monitoring-the-cluster-upgrade.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/day_2_core_cnf_clusters/upgrading/telco-upgrade-completing-the-upgrade.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="telco-upgrade-monitoring-the-cluster-upgrade_{context}"]
+= Monitoring the cluster upgrade
+
+You can monitor the cluster upgrade.
```

Comment:
I know this seems pretty straightforward, but maybe try to give a little more detail here on what you're monitoring, or what you're monitoring for.